src/com/google/common/css/compiler/ast/GssParserCC.jj

```diff
@@ -707,6 +707,8 @@ PARSER_END(GssParserCC)
   // Special handling needed because ':not' takes simple selectors as
```

Comment:
Should we just generalize this to two tokens: `SIMPLE_SELECTOR_FUNCTION` and `SELECTOR_FUNCTION`? It seems like a bunch of duplicate code in the `pseudo` expansion would go away.